src/b-splines/quadratic.jl

```diff
@@ -129,6 +130,8 @@ function prefiltering_system{T,TC,BC<:Union{Flat,Reflect}}(::Type{T}, ::Type{TC}
     Woodbury(lufact!(Tridiagonal(dl, d, du), Val{false}), rowspec, valspec, colspec), zeros(TC, n)
 end
 
+# quadratic and cubic share the same woodbury correction for `Line` type. No need
+# to re-implement
```

Comment:
What does this comment add here? This idea is expressed in the relevant docstrings for cubic interpolation, so I think this comment can be removed entirely.